account_partner_reconcile/models/res_partner.py

```diff
@@ -9,19 +9,23 @@ class ResPartner(models.Model):
 
     def action_open_reconcile(self):
         # Open reconciliation view for customers and suppliers
+        self.ensure_one()
+
+        if not self:
+            return {}
```

Comment:
self.ensure_one already checks that there is at least one record

Reply:
you are right. done as requested.